pwndbg/commands/xor.py

```diff
@@ -31,10 +31,17 @@ def xor(address, key, count):
     XOR ``count`` bytes at ``address`` with the key ``key``.
     '''
     if not isinstance(address, six.integer_types):
-        address = int(address, 16)
+        try:
+            address = int(address, 0)
+        except ValueError:
+            print('Invalid address %s' % address)
+            return
 
-    xorred_memory = xor_memory(address, key, count)
-    pwndbg.memory.write(address, xorred_memory)
+    try:
+        xorred_memory = xor_memory(address, key, count)
+        pwndbg.memory.write(address, xorred_memory)
+    except gdb.MemoryError as e:
+        print(e)
 
 @pwndbg.commands.Command
```

Comment:
The only reason we have to deal with `int`'ing a thing is because this is a `Command` instead of a `ParsedCommand`.  If you change it, we should be getting `gdb.Value` objects, which are much nicer.